specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/CopyModel.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
could you add an example of what the request to the target endpoint would look like? I'm talking about the call to be done before this one, reserving the modelId in the target resource

Reply:
I've shared an actual example with you offline. Pls let me know if that works for you.

Reply:
Would you be willing to share this with me as well?